pymc/tests/test_distributions_moments.py

```diff
@@ -1010,8 +1011,8 @@ def test_moyal_moment(mu, sigma, size, expected):
         (2, rand1d, np.eye(2), 2, np.full((2, 2), rand1d)),
         (2, rand1d, np.eye(2), (2, 5), np.full((2, 5, 2), rand1d)),
         (2, rand2d, np.eye(3), None, rand2d),
-        (2, rand2d, np.eye(3), 2, np.full((2, 2, 3), rand2d)),
-        (2, rand2d, np.eye(3), (2, 5), np.full((2, 5, 2, 3), rand2d)),
+        (2, rand2d, np.eye(3), 2, np.full((2, 3), rand2d)),
+        (2, rand2d, np.eye(3), (2, 5, 2), np.full((2, 5, 2, 3), rand2d)),
```

Comment:
These two used to assert the inconsistent interpretation where size was concatenated in front of all other dimensions for multivariate RVs.